frontend/src/app/offers/offer-join-form/offer-join-form.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <form [formGroup]="joinForm" (ngSubmit)="onSubmit()">
+    <div class="form-group row">
+      <label class="col-form-label" for="{{ email }}">Email:</label>
+        <div class="col-md-5">
+          <input disabled class="form-control" formControlName="applicant_email" id="{{ email }}"/>
```

Comment:
You use {{ }} to bind to `email` variable, that is not defined in the component, so as a result, you get empty for attributes in HTML (same problem for all other labels). Define them just using strings, and if you define `for` attribute in a `label` don't forget to define `id' in `input`, so focus can be set after clicking on label.